lib/membrane_rtc_engine/endpoints/webrtc/rtp_connection_allocator.ex

```diff
@@ -113,14 +113,16 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPConnectionAllocator do
     estimation_increasing? =
       state.available_bandwidth == :unknown or estimation >= state.available_bandwidth
 
+    overuse_allowed? = estimation_increasing? or state.prober_status != :allowed_overuse
```

Comment:
You sure we want to have `state.prober_status != :allowed_overuse` and not `state.prober_status == :allowed_overuse`?

Reply:
Yes. If we used `state.prober_status == :allowed_overuse`, then we would never actually leave `:allowed_overuse` unless we reached `:allocated_bandwidth`. This is problematic when `:allocated_bandwidth` is higher than your bandwidth, because it will never happen. Long story short: we would never decrease the quality.